packages/placement-ordering/src/ordering.js

```diff
@@ -119,7 +120,7 @@ function buildTiles(choices, response, outcomes, opts) {
   }
 }
 
-export function buildState(choices, response, outcomes, opts) {
+export function buildState(choices, response, outcomes, opts, removeTile) {
```

Comment:
remove `removeTile`, add a boolean to the opts param: `opts: { includeTarget: boolean, allowSameChoiceInTargets: boolean }`, also use the language of the api 'choices' and 'targets' - 'tiles' is an internal implementation name.